exchanges/hitbtc/hitbtc_websocket.go

```diff
@@ -838,3 +807,50 @@ func (h *HitBTC) wsGetTrades(c currency.Pair, limit int64, sort, by string) (*Ws
 	}
 	return &response, nil
 }
+
+// subToReq returns the subscription as a map to populate WsRequest
+func subToReq(s *subscription.Subscription, maybePair ...currency.Pair) *WsRequest {
```

Comment:
[Bithumb](https://github.com/thrasher-corp/gocryptotrader/pull/1618/files#diff-4d15fff6eb93a58ccc68b3ed41eec1a75dc0b3bb5bd2daba9dc7f1d2b2aa1e0aR96) got a test for this func, I request one here too if possible

Reply:
Fixed https://github.com/gbjk/gocryptotrader/commit/1bd1ef2dc